src/components/Tracks/Tracks.js

```diff
@@ -1,28 +1,45 @@
-import React from 'react';
-import Card from '../Card/Card';
-import '../Tracks/Tracks.css';
+import React from "react";
+import Card from "../Card/Card";
+import MusicGenreRow from "../MusicGenreRow/MusicGenreRow";
+import MusicCardRow from "../MusicCardRow/MusicCardRow";
+import "../Tracks/Tracks.css";
 
 const Tracks = ({ music }) => {
-    const musicCards = music.map(song => {
-        return(
-           <Card 
-                id={song.id}
-                key={song.id}
-                coverArt={song.coverArt}
-                artist={song.artist}
-                genre={song.genre}
-                title={song.title}
-                audioFile={song.audioFile}
-            />
-           
-        );
-    });
+  const rows = [];
```

Comment:
Hmm - are you storing data in this array? best to create a piece of state to hold it then, rather than just having it hang out!